exporter/diagnostic_data_collector.go

```diff
@@ -43,6 +43,11 @@ func (d *diagnosticDataCollector) Collect(ch chan<- prometheus.Metric) {
 
 	cmd := bson.D{{Key: "getDiagnosticData", Value: "1"}}
 	res := d.client.Database("admin").RunCommand(d.ctx, cmd)
+	if res.Err() != nil {
+		if isArbiter, _ := isArbiter(d.ctx, d.client); isArbiter {
+			return
```

Comment:
How about we log that we hit an arbiter?

Reply:
No. This runs every 1 minute and we would fill the logs.

Reply:
Got it.

Reply:
What about logging to Debug level?